sessionctx/variable/sysvar.go

```diff
@@ -852,10 +852,6 @@ var defaultSysVars = []*SysVar{
 		s.EnablePointGetCache = TiDBOptOn(val)
 		return nil
 	}},
-	{Scope: ScopeGlobal, Name: TiDBEnableAlterPlacement, Value: BoolToOnOff(DefTiDBEnableAlterPlacement), Hidden: true, Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
-		s.EnableAlterPlacement = TiDBOptOn(val)
-		return nil
-	}},
```

Comment:
When you remove this from here, can you add it to https://github.com/pingcap/tidb/blob/master/sessionctx/variable/removed.go

This has some special semantics to make applications not break as much. Otherwise LGTM

Reply:
Addressed comments. PTAL again @morgo